### PR TITLE
Add descriptions to changelog entries

### PR DESCRIPTION
**Pull request by tmattio**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -6,25 +6,134 @@
...
+
 - Add opam users (ocaml/ood#80, by @tmattio)
+
+  This allows us to display the package authors and maintainers with an avatar and add links
+  their GitHub profiles from the packages overviews.
+
 - Remove french tutorials (ocaml/ood#79, by @tmattio)
+
+  For the most part the French translations of the tutorials in ocaml.org are either incomplete or outdated.
+  In their current state, we won't import the translations, and we will work on improving them instead instead.
```
Comment (tmattio) [inline CHANGES.md]:
```suggestion
  In their current state, we won't import the translations, and we will work on improving them instead.
```

File: CHANGES.md [hunk 2]
```diff
@@ -6,25 +6,134 @@
...
+  The library (hilite) reuses the textmate grammars from vscode with some minor modifications because
+  the textmate library wasn't happy parsing them.
+
 - Add initial set of Best Known Methods (#107, by @tmattio)
+
+  Added the Best Known Methods data and included them on the Best Practices page.
+
 - Add initial toplevel to homepage (#106, by @tmattio)
+
+  Add a toplevel in the homage that provides function from the Standard Library.
```
Comment (patricoferris) [inline CHANGES.md]:
```suggestion
  Add a toplevel in the homepage that provides function from the Standard Library.
```

File: CHANGES.md [hunk 3]
```diff
@@ -6,25 +6,134 @@
...
+
 - Remove french tutorials (ocaml/ood#79, by @tmattio)
+
+  For the most part the French translations of the tutorials in ocaml.org are either incomplete or outdated.
+  In their current state, we won't import the translations, and we will work on improving them instead instead.
+
 - Import releases (ocaml/ood#78, by @tmattio)
 
+  Import the releases data from the current ocaml.org website.
+  The data is imported as-is, which lots of broken links and images.
```
Comment (patricoferris) [inline CHANGES.md]:
```suggestion
  The data is imported as-is, with lots of broken links and images.
```

**Review by patricoferris — APPROVED** [review]
Thanks for doing this :))

**Pull request merged**